jquery.overlay.js

```diff
@@ -4,11 +4,13 @@
  * Repository: https://github.com/yuku-t/jquery-textoverlay
  * License:          MIT
  * Original Author:  Yuku Takahashi
- * Modifying Author: Niki Castle
+ * Modifying Authors: Niki Castle, Nick Willett-Jeffries
  *
- * This version is from https://github.com/neekee/jquery-overlay/tree/master, which includes
- * modifications made to get this working for input fields and resizeable textareas, and to
- * allow nested matching.
+ * This version is from https://github.com/nwj/jquery-overlay/tree/master, which introduces support for
```

Comment:
The css stuff came in from yuku-t's upstream. See this commit: https://github.com/yuku-t/jquery-overlay/commit/67bc6dddaf4fe66471525fa7d0b8f701c95d9791.

I think it's possible to mess up alignment through those styles and there's no whitelist currently for what styles are valid.